library/cwm/src/lib/cwm/common_widgets.rb

```diff
@@ -271,6 +271,8 @@ def cwm_definition
 
   # A selection of a value via radio buttons.
   # The {#label} method is mandatory.
+  # @note if radio buttons are modified during runtime, like with #change_items
+  #   then handle won't work correctly unless handle_all_events specified
```

Comment:
Please have a changes entry for this. "Fix handl**ing** events for radio buttons (that were changed with change_items ?)"